packages/manager/apps/dedicated/client/app/dedicated/server/server.component.js

```diff
@@ -0,0 +1,20 @@
+import controller from './server.controller';
+import template from './server.html';
+
+export const component = {
+  bindings: {
+    ola: '<',
+    orderPrivateBandwidthLink: '<',
+    orderPublicBandwidthLink: '<',
+    resiliatePrivateBandwidthLink: '<',
+    resiliatePublicBandwidthLink: '<',
+    server: '<',
+    specifications: '<',
+  },
+  controller,
+  template,
+};
+
+angular.module('App').component('dedicatedServer', component);
```

Comment:
Maybe all angular declarations could be grouped in the angular module definition